src/lib/themes-test.js

```diff
@@ -93,6 +93,30 @@ describe("lib/themes", () => {
       // Input object should not have been changed.
       expect(themeInput).to.deep.equal(deprecatedTheme);
     });
+
+    it("should include advanced colors themes", () => {
```

Comment:
To verify that the test below is not just passing due to blindly accepting any color property, could you add a test that includes an unsupported property name, and verify the resulting behavior?